test_accounts/appeals_apis/appeals_status_test_accounts.md

```diff
@@ -0,0 +1,45 @@
+# Sandbox Test Accounts for Appeals Status API (v1)
+
+This document provides test user accounts for the [Appeals Status API (v1)](https://developer.va.gov/explore/api/appeals-status/docs?version=current). The API will return synthetic VA records for all test accounts.
+
+**Do not modify any of the attributes for these test accounts, including passwords.** If you need to use one of these accounts for a different use case, [contact us](https://developer.va.gov/support/contact-us) on our developer portal.
+
+## Authentication and Authorization
+
+The authentication model for the [Appeals Status API (v1)](https://developer.va.gov/explore/api/appeals-status/docs?version=current) uses OAuth 2.0/OpenID Connect. The following authorization models are supported:
+- [Authorization code flow](https://developer.va.gov/explore/api/appeals-status/authorization-code)
+- [Client Credentials Grant (CCG)](https://developer.va.gov/explore/api/appeals-status/client-credentials)
+
+### Logging in with ID.me
+
+To log in to the sandbox environment using ID.me, make sure you:
+
+1. Choose an account that is specific to the scenario you want to test.
+2. Select **ID.me** to sign in to the sandbox environment.
+3. Enter the ID.me username and password. The password for all ID.me test accounts is: **Password1234!**
+4. **Don’t change any preselected answers** when asked about receiving an authentication code. Just click “Continue” to go to the next step.
+
+### Logging in with Login.gov
+
+To log in to the sandbox environment using Login.gov, make sure you:
+
+1. Choose an account that is specific to the scenario you want to test.
+2. Select **Login.gov** to sign in to the sandbox environment.
+3. Enter the Login.gov username and password. The password for all Login.gov test accounts is: **Password12345!!!**
+4. Use the Login.gov MFA seed to generate a 2FA code with an app such as Google Authenticator or Authy.
```

Comment:
These sections seemed important to continue to include for consumers using the Authorization Code Grant authorization method. I placed these as sub-headings under "Authentication and Authorization."